composer.json

```diff
@@ -13,50 +13,59 @@
     ],
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": "^7.2",
+        "php": "^8.1",
```

Comment:
Is a minimum of PHP 8.1 really required? According to core support, we should begin with PHP 7.4 up to 8.3. Or is something inside the code preventing PHP 7.4?